commcare_connect/users/models.py

```diff
@@ -46,3 +46,12 @@ def get_absolute_url(self) -> str:
 
         """
         return reverse("users:detail", kwargs={"pk": self.id})
+
+
+class ConnectIDUserLink(models.Model):
```

Comment:
I didn't notice before but I think there should be a unique constraint on `user + commcare_username` with error handling in the view (perhaps to just ignore the error if the link already exists or adjust the return code (201 created vs 200).